src/app/client/src/app/modules/search/components/user-profile/user-profile.component.html

```diff
@@ -188,7 +188,7 @@ <h5 class="ui right floated basic icon circular button" [queryParams]="queryPara
                   </div>
                 </div>
               </div>
-              <div *ngIf="badge.createdTS || badge.createdTs" class="meta">{{badge.createdTS || badge.createdTs | dateFormat}}</div>
+              <div class="meta">{{badge.createdDate || badge.createdTs | dateFormat: 'MMM YYYY'}}</div>
```

Comment:
Not as per latest change in angular-migration branch.

Expected:
< div *ngIf="badge.createdTS || badge.createdTs" class="meta" >{{badge.createdTS || badge.createdTs | dateFormat}}< /div >